src/components/app/list-new/AppListService.ts

```diff
@@ -133,9 +133,14 @@ export const getInitData = (payloadParsedFromUrl : any, serverMode : string): Pr
         // set list data for env cluster & namespace
         const environmentClusterAppListData = new Map()
         const clusterMap = new Map()
+        const projectMap = new Map()
         for (const cluster of clusterList) {
             clusterMap.set(cluster.id, cluster.cluster_name)
         }
+        for (const project of projectList) {
```

Comment:
All clusterList, projectList, and environmentList should have a fail-safe check here as these might be undefined.